send_multiple_messages.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import sys
+import threading
+
+from client import send_data
+
+
+def send_messages(server_ip, server_port, messages):
```

Comment:
Why do we need to support this behavior?
The client isn't running multiple connections, seems a bit of an overkill to open a new thread for each message it wants to send.

Reply:
This is a test script to check that the server is handling connections in parallel. A new thread (and a new connection) is created for each message so all the messages will be sent in parallel.